airlock_processor/tests/test_status_change_queue_trigger.py

```diff
@@ -1,6 +1,6 @@
 from json import JSONDecodeError
 import os
-import unittest
+import pytest
 from unittest import mock
 from unittest.mock import MagicMock, patch
```

Comment:
should this just be 
```suggestion
from mock import MagicMock, patch
```
like you/we do elsewhere